torch/_torch_docs.py

```diff
@@ -1209,6 +1209,42 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.tensor_split,
+           r"""
+tensor_split(input, indices_or_sections, dim=0) -> List of Tensors
+
+Split a tensor into multiple sub-tensors, all of which are views of the
+input tensor. This function is based on :func:`numpy.array_split`.
+
+Args:
+    input (Tensor): the tensor to split
+    indices_or_sections (int or (list(int))):
+        If :attr:`indices_or_sections` is an integer, it represents the number of
+        sections to divide the input into. The input does not have to be divisible
+        by the number of sections. The first :code:`int(input.size(dim) % sections)`
+        output tensors will have have size :code:`int(input.size(dim) / sections) + 1`
+        in dimension :attr:`dim`. The rest will have size
+        :code:`int(input.size(dim) / sections)`.
+
+        If :attr:`indices_or_sections` is a list of ints, it represents a list of
```

Comment:
"It represents..." -> "then :attr:\`input\` is split along dimension :attr:\`dim\` at offsets..." ?

It's important to express that the splitting is along dim, and the docs should tie the list of ints directly to their effect instead of describing what they represent.